### PR TITLE
[Cross-PR] Optimize new images

### PR DESCRIPTION
**Pull request by alexfornuto**
Optimizes the newly introduced images in #18979 for the web.

### CODE REVIEW

**Comment by ptgott** [comment]
@alexfornuto What command do you use to optimize images? At some point, we'll probably want to validate image sizes in our docs CI pipeline and, if an image is too large, print that command as a suggested way to reduce its size.

**Comment by alexfornuto** [comment]
@ptgott I use [Trimage](https://github.com/Kilian/Trimage). It's cross-platform between macOS and 'nix, and has a CLI version, so we could consider building it into our processes. The way the docs repo runs lint on commit, we could do the same with Trimage maybe?

**Review by zmb3 — APPROVED** [review]

**Comment by ptgott** [comment]
@alexfornuto This is probably because I was changing around images while responding to PR feedback, though I'm not sure why _all_ the images changed. In any case, I think it would make sense to rerun the optimization command once we've got the signoff  to merge the main PR

**Pull request closed without merging**
